classes/services/queryBuilders/BaseQueryBuilder.inc.php

```diff
@@ -15,52 +15,10 @@
 
 namespace PKP\Services\QueryBuilders;
 
-use Illuminate\Database\Capsule\Manager as Capsule;
```

Comment:
I suppose the `BaseQueryBuilder` isn't needed anymore? Or did you have a use in mind for it?

Reply:
Agreed, I'll remove it.